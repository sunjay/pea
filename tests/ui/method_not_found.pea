fn main() {
    let x = b"abc";
    x.foo();
}
