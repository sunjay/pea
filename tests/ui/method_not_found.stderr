[tests/ui/method_not_found.pea:3:7-3:9] error: no method named `foo` found for type `[u8]` in the current scope

error: aborting due to 1 previous error

